pages/docs/reference/building-mpp-with-gradle.md

```diff
@@ -1721,6 +1724,74 @@ It is important to note that some of the [Kotlin/Native targets](#supported-plat
 A target that is not supported by the current host is ignored during build and therefore not published. A library author may want to set up
 builds and publishing from different hosts as required by the library target platforms.
 
+### Target shortcuts
+
+Some native targets are often created together and use the same sources. For example building for iOS device and simulator
+is represented by different targets (`iosArm64` and `iosX64` respectively) but usually a source code for device and simulator
+doesn't differ. A canonical way to express such shared code in the multiplatform project model is to create an intermediate
+source set `iosMain` and configure links between it and platform source sets:
```

Comment:
The entire paragraph with my suggestions:
Some native targets are often created together and use the same sources. For example, building for an iOS device and a simulator is represented by different targets (`iosArm64` and `iosX64` respectively) but their source codes are usually the same. A canonical way to express such shared code in the multiplatform project model is creating an intermediate source set (`iosMain`) and configuring links between it and the platform source sets: